### ISSUE TITLE
[Failed-AutoCherryPick] - Add virt who config cases to All Tiers Upgrade Tests

### ISSUE THREAD

**Issue by Satellite-QE** [SatelliteQE/robottelo#11151]
#### Auto-Cherry-Pick WorkFlow Failure:
- To Branch: 6.13.z
- [Failed Cherrypick Action](https://github.com/SatelliteQE/robottelo/actions/runs/4608186753)
- [Parent Pull Request](https://github.com/SatelliteQE/robottelo/pull/11131)

**Comment by ogajduse** [comment]
@yanpliu could you please cherry-pick #11131 to 6.13.z branch?

**Comment by yanpliu** [comment]
> @yanpliu could you please cherry-pick #11131 to 6.13.z branch?
@ogajduse when I cherry-pick the commits https://github.com/SatelliteQE/robottelo/pull/11131/commits/3cbee71dd9224e2c7a03cb0b2b5b9b953614fa93

it shows "fatal: bad object 3cbee71dd9224e2c7a03cb0b2b5b9b953614fa93"

```
(robottelo_vv) [virtwho@dell-per740-68-vm-04 robottelo]$ git fetch --all
Fetching origin
remote: Enumerating objects: 3783, done.
remote: Counting objects: 100% (250/250), done.
remote: Compressing objects: 100% (15/15), done.
remote: Total 3783 (delta 237), reused 244 (delta 235), pack-reused 3533
Receiving objects: 100% (3783/3783), 1014.41 KiB | 1.52 MiB/s, done.
Resolving deltas: 100% (2299/2299), completed with 113 local objects.
From github.com:yanpliu/robottelo
 + 150208256...c7f2a6759 master                                           -> origin/master  (forced update)
 + fd4f34040...654ac8d06 6.10.z                                           -> origin/6.10.z  (forced update)
 * [new branch]          6.11-fails                                       -> origin/6.11-fails
 + b87a14f51...d77435218 6.9.z                                            -> origin/6.9.z  (forced update)
 * [new branch]          JacobCallahan-patch-1                            -> origin/JacobCallahan-patch-1
 * [new branch]          JacobCallahan-pyupboot                           -> origin/JacobCallahan-pyupboot
 * [new branch]          JacobCallahan-pyupbot                            -> origin/JacobCallahan-pyupbot
......
 * [new branch]          virt-who-plugin-setup-teardown-support           -> origin/virt-who-plugin-setup-teardown-support
Fetching upstream
(robottelo_vv) [virtwho@dell-per740-68-vm-04 robottelo]$  git checkout -b cpick-11131 upstream/6.13.z
Branch 'cpick-11131' set up to track remote branch '6.13.z' from 'upstream'.
Switched to a new branch 'cpick-11131'
(robottelo_vv) [virtwho@dell-per740-68-vm-04 robottelo]$ git cherry-pick -x 3cbee71dd9224e2c7a03cb0b2b5b9b953614fa93
fatal: bad object 3cbee71dd9224e2c7a03cb0b2b5b9b953614fa93

```
Could you please give me some help about this? Thank you very much.

**Comment by ogajduse** [comment]
@yanpliu Your original commit 3cbee71 was merged as ecaf81c. Could you try to cherry-pick ecaf81c, I believe that this one should work.

![image](https://github.com/SatelliteQE/robottelo/assets/20440883/00499642-8766-4dd3-800f-fec88a15f250)
